docker/unittest-dedb/setup-dev-database.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -x
+set -e
+
+#psql -h 127.0.0.1 -U $POSTGRES_USER -d postgres -c "CREATE DATABASE de WITH OWNER $POSTGRES_USER;"
```

Comment:
This may not ever be a problem, but in the case that `$POSTGRES_USER` is not equal to `de`, the `de` database will never be created. I didn't see a way to create a database only if it doesn't exist already. Another option might be to use `createuser --no-createdb` command to create the user, so that the command to create the database can always be executed without causing an error.